code/Test_Definitions/NumberVerification_device_phone_number_share.feature

```diff
@@ -5,7 +5,7 @@ Feature: Camara Number Verification API device phone number share
 
 # Input to be provided by the implementation to the tests
 # References to OAS spec schemas refer to schemas specified in
-# https://raw.githubusercontent.com/camaraproject/NumberVerification/main/code/API_definitions/number_verification.yaml
+# https://raw.githubusercontent.com/camaraproject/NumberVerification/main/code/API_definitions/number-verification.yaml
```

Comment:
Is somewhere defined which version of the API definition the .feature file refers to? This link goes to main

Maybe a relative link would be better to stay within the tag/branch, but I don't this is possibly for raw content. An alternative would be to link to `/code/API_definitions/number_verification.yaml`

Reply:
I've applied your suggestion.